src/requireCssModule.js

```diff
@@ -18,19 +18,57 @@ import type {
   StyleModuleMapType
 } from './types';
 
-const getTokens = (runner, cssSourceFilePath: string, filetypes): StyleModuleMapType => {
+type FileTypeOptions = {|
+  syntax: string,
+  plugins: Array<string>
+|};
+
+const getFiletypeOptions = (cssSourceFilePath: string, filetypes: Object): ?(string|FileTypeOptions) => {
   const extension = cssSourceFilePath.substr(cssSourceFilePath.lastIndexOf('.'));
-  const syntax = filetypes[extension];
+  const filetype = filetypes ? filetypes[extension] : null;
 
-  const options: Object = {
-    from: cssSourceFilePath
-  };
+  return filetype;
+};
+
+const getSyntax = (filetypeOptions: ?(string|FileTypeOptions)) => {
```

Comment:
Require that `filetypeOptions` is always `FileTypeOptions`. Do not call `getSyntax` if `filetypeOptions` option is void.